docs-2.0/2.quick-start/4.nebula-graph-crud.md

```diff
@@ -72,14 +72,19 @@ To make sure the follow-up operations work as expected, take one of the followin
 * Create a graph space:
 
     ```ngql
-    CREATE SPACE <graph_space_name> [(partition_num=<partition number>, replica_factor=<replica number>, vid_type=fixed_string(<string_number>))]
+    CREATE SPACE [IF NOT EXISTS] <graph_space_name>
+        [(partition_num = <partition_number>, 
+        replica_factor = <replica_number>, 
+        charset = <charset>, 
+        collate = <collate>, 
+        vid_type = {FIXED_STRING(<N>)) | INT64}];
     ```
 
     | Property       | Description                                                                                                                                                                                                                        |
     | -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
     | partition_num  | Specifies the number of partitions in each replica. The suggested number is the number of hard disks in the cluster times 5. For example, if you have 3 hard disks in the cluster, we recommend that you set 15 partitions.           |
     | replica_factor | Specifies the number of replicas in the Nebula Graph cluster. The suggested number is 3 in a production environment and 1 in a test environment. The replica number must always be an **odd** number for the need of quorum-based voting. |
-    |vid_type | To insert a vertex with a string-typed VID, set the `fixed_string` length for the maximum VID length. The default `fixed_string` is 8, and recommend to be less than 64. `fix_string(N)` is similar to `char(N)` in SQL. |
+    |vid_type | Specifies the data type of VIDs in a graph space. Available values are `FIXED_STRING(N)` and `INT64`. `N` represents the maximum length of the VIDs and it must be a positive integer. The default value is `FIXED_STRING(8)`. If you set a VID length greater than `N`, Nebula Graph throws an error. To set the integer VID for vertices, set `vid_type` to `INT64`. |
```

Comment:
can you can some <br> for pretty reading?

Reply:
MD tables are clumsy. I need HTML to optimize it.